### PR TITLE
Revert to readcounter

### PR DESCRIPTION
**Pull request by lpoli**
## Changes
This PR reverts ReadMarker changes from read size to read counter
## Tests 
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber: https://github.com/0chain/blobber/pull/632
- gosdk: https://github.com/0chain/gosdk/pull/429
- system_test:
- zboxcli: https://github.com/0chain/zboxcli/pull/211
- zwalletcli:
- Other: ...

### CODE REVIEW

**Review by peterlimg — APPROVED** [review]
lgtm

File: code/go/0chain.net/smartcontract/storagesc/blobber.go [hunk 1]
```diff
@@ -404,13 +420,16 @@ func (sc *StorageSmartContract) commitBlobberRead(t *transaction.Transaction,
 			"error fetching blobber object")
 	}
 
+	const CHUNK_SIZE = 64 * KB
```
Comment (Sriep) [inline code/go/0chain.net/smartcontract/storagesc/blobber.go:423]: This is declared twice, here and in[ blober2_test.go](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/storagesc/blobber2_test.go#L25). I suggest merging the two.

**Review by Sriep — APPROVED** [review]

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | fix/revert-to-readcounter  |
| zwalletcli  | staging  |
| 0chain  | fix/revert-to-readcounter  |
| blobber  | fix/revert-to-readcounter  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by peterlimg** [comment]
@lpoli please resolve the system test errors, all revert PRs should be closed soon.

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | fix/revert-to-readcounter  |
| zboxcli  | fix/revert-to-readcounter  |
| zwalletcli  | staging  |
| 0chain  | fix/revert-to-readcounter  |
| blobber  | fix/revert-to-readcounter  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/revert-to-readcounter  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | fix/revert-to-readcounter  |
| zwalletcli  | staging  |
| 0chain  | fix/revert-to-readcounter  |
| blobber  | fix/revert-to-readcounter  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/revert-to-readcounter  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | fix/revert-to-readcounter  |
| zwalletcli  | staging  |
| 0chain  | fix/revert-to-readcounter  |
| blobber  | fix/revert-to-readcounter  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**